templates/single-quiz/pagination.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * The template for displaying the quiz pagination.
+ *
+ * @author   Automattic
+ * @package  Sensei
+ * @category Templates
+ * @version  3.15.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+global $sensei_question_loop;
+
+?>
+
+<div class="sensei-quiz-pagination">
+	<div class="sensei-quiz-pagination__list">
+		<?php
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- No need to escape the pagination links.
+		echo paginate_links(
+			/**
+			 * Filters the quiz questions paginate links args.
+			 *
+			 * @see https://developer.wordpress.org/reference/functions/paginate_links/
+			 *
+			 * @since 3.15.0
+			 *
+			 * @param array $args The pagination arguments.
+			 */
+			apply_filters(
+				'sensei_quiz_pagination_args',
+				[
+					'total'     => $sensei_question_loop['total_pages'],
+					'current'   => $sensei_question_loop['current_page'],
+					'format'    => '?quiz-page=%#%',
+					'type'      => 'list',
+					'mid_size'  => 1,
+					'prev_next' => false,
+				]
+			)
+		);
+		?>
+	</div>
+
+	<div class="sensei-quiz-pagination__actions">
+		<?php if ( Sensei_Quiz::is_reset_allowed( Sensei()->quiz->get_lesson_id( get_the_ID() ) ) ) : ?>
+			<div class="sensei-quiz-pagination__button-link">
+				<button type="submit" name="quiz_reset" class="sensei-stop-double-submission has-background has-text-color">
```

Comment:
I'm not too clear if `has-background` and `has-text-color` are needed here. I think they pull in the theme's colors or something like that? Would you mind clarifying?

Reply:
The initial issue here was that some of the `twenty` themes have very specific CSS selectors for the button background color that can't be easily overwritten.

By using these classes I've managed to avoid the use of `!important` or very long selectors to overwrite the styles. 

But thinking about it now, adding theme-specific classes doesn't make much sense. I've removed the classes and changed to code to use `!important` in 8ae06031cb8b0b4e1883203ad0e9a8638ff96081. Let me know if there is a better approach.

Reply:
I tested this with Twenty Twenty One. The problem appears to be that the theme's CSS selector (`button:not(:hover):not(:active):not(.has-background)`) has much higher specificity than our selector because of all the pseudo classes. Probably the easiest way to combat this would be to give the button an ID like `sensei-quiz-save` and then use that to style it.

Reply:
@donnapep Yeah, that's an option for the reset/save buttons, but we will have the same problem down the road (#4521) when the pagination links are transformed into buttons. Should I add the IDs now and figure this out later?

Reply:
I think this update https://github.com/Automattic/sensei/commit/70caded9fbc035026d2ce092cfae6d075e28c489 solves all of the above.